app/cdap/components/DirectiveInput/IconStore/CrossIcon.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+
+export const CrossIcon = (
+  <svg
+    width="18"
+    height="18"
```

Comment:
can we use mui icon for close icon plz
plz use form mui v4
smriti has already used one
plz check with her and use it here

we can delete this file as well

Reply:
Done